projects/gateway2/deployer/deployer.go

```diff
@@ -193,25 +166,36 @@ func (d *Deployer) Render(ctx context.Context, name, ns string, vals map[string]
 		return nil, fmt.Errorf("failed to render helm chart: %w", err)
 	}
 
-	objs, err := ConvertYAMLToObjects(d.scheme, []byte(release.Manifest))
+	objs, err := ConvertYAMLToObjects(d.cli.Scheme(), []byte(release.Manifest))
 	if err != nil {
 		return nil, fmt.Errorf("failed to convert yaml to objects: %w", err)
 	}
 	return objs, nil
 }
 
 func (d *Deployer) GetObjsToDeploy(ctx context.Context, gw *api.Gateway) ([]client.Object, error) {
-	objs, err := d.renderChartToObjects(ctx, gw)
+	logger := log.FromContext(ctx)
+
+	vals, err := d.getValues(ctx, gw)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get values to render objects: %w", err)
+	}
+	logger.V(1).Info("got deployer helm values", "values", vals)
```

Comment:
[nit] Can we log which gateway here as well? 

```suggestion
	logger.V(1).Info(fmt.Sprintf("got deployer helm values for gateway %s.%s", gw.Name, gw.Namespace), "values", vals)
```

Reply:
added in d6c419d